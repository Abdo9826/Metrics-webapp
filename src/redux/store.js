import { configureStore } from '@reduxjs/toolkit';
import { combineReducers } from 'redux';
import thunk from 'redux-thunk';
import logger from 'redux-logger';
import dataReducer from './data';

const rootReducer = combineReducers({
  data: dataReducer,
});

const store = configureStore({
  reducer: rootReducer,
  middleware: [thunk, logger],
});

export default store;
