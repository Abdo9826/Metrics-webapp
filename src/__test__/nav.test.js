import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navbar from '../components/navbar';
import store from '../redux/store';

it('To check test is working', () => {
  expect(true).toBe(true);
});

it('Test if header render correctly', () => {
  const TREE = renderer.create(
    <Provider store={store}>
      <BrowserRouter>
        <Navbar />
      </BrowserRouter>
    </Provider>,
  );
  expect(TREE).toMatchSnapshot();
});
