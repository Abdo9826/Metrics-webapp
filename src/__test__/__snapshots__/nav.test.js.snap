// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if header render correctly 1`] = `
<nav>
  <a
    className="white"
    href="/"
    onClick={[Function]}
  >
    <h3>
       __
    </h3>
  </a>
  <h1>
    All conturies
  </h1>
</nav>
`;
